client-data.md

```diff
@@ -1,21 +1,17 @@
 # Filecoin Client Data Processing
```

Comment:
Do we have to repeat filecoin in a subsection of the... filecoin specs ? My philosophy is to get to the point: the less words the better (less cognitive load)

Reply:
ack